mobile/src/main/java/com/opendashcam/Util.java

```diff
@@ -38,9 +38,9 @@
     private static final String NOTIFICATIONS_CHANNEL_ID_MAIN_NOTIFICATIONS = "1001";
     private static final String NOTIFICATIONS_CHANNEL_NAME_MAIN_NOTIFICATIONS = "Main notifications";
 
-    private static int QUOTA = 1000; // megabytes
-    private static int QUOTA_WARNING_THRESHOLD = 200; // megabytes
-    private static int MAX_DURATION = 45000; // 45 seconds
+    private static int QUOTA = 3000; // megabytes
+    private static int QUOTA_WARNING_THRESHOLD = 500; // megabytes
+    private static int MAX_DURATION = 600000; // 10 minutes
```

Comment:
This is the duration per video clip.
Some devices may have trouble recording longer video clips. For example, https://9to5google.com/2021/08/30/pixel-5a-overheating-issues-google/
> Pixel 5a would stop recording after about 4 minutes of recording 4K video with heat as the cause

thus we keep the individual snippets as short as possible.